stestr/commands/list.py

```diff
@@ -66,15 +81,20 @@ def take_action(self, parsed_args):
                             test_path=self.app_args.test_path,
                             top_dir=self.app_args.top_dir,
                             blacklist_file=args.blacklist_file,
+                            exclusion_list_file=args.exclusion_list_file,
                             whitelist_file=args.whitelist_file,
+                            inclusion_list_file=args.inclusion_list_file,
                             black_regex=args.black_regex,
+                            exclusion_regex=args.exclusion_regex,
                             filters=filters)
 
 
 def list_command(config='.stestr.conf', repo_type='file', repo_url=None,
                  test_path=None, top_dir=None, group_regex=None,
-                 blacklist_file=None, whitelist_file=None, black_regex=None,
-                 filters=None, stdout=sys.stdout):
+                 blacklist_file=None, exclusion_list_file=None,
+                 whitelist_file=None, inclusion_list_file=None,
```

Comment:
If we're using `--include-list`  and `--exclude-list` I think the Python API kwargs should match that name and be `include_list` and `exclude_list` just to make it a bit clearer.

Reply:
:+1: